src/commands/object/create.js

```diff
@@ -0,0 +1,82 @@
+const { Command, flags } = require('@oclif/command');
+
+const Config = require('../../utils/config');
+const Request = require('../../utils/request');
+
+/* Class representing the Create command */
+class CreateCommand extends Command {
+  /**
+   * Create an instance of CreateCommand class
+   */
+  constructor() {
+    super(...arguments);
+    this._config = new Config();
+    const { baseUrl, username, password } = this._config.load();
+    this._request = new Request({ baseUrl, username, password });
+  }
+
+  /**
+   * Read data from the stdin
+   * @private
+   * @returns {Promise<string>} Promise representing the data string read from stdin
+   */
+  async _readStdin() {
```

Comment:
Do we need to clean up listeners here on error and end, so we don't end up attaching new listeners every call? Right now this is only called once each run, but future implementation might just re-use this functionality as they see fit.

Reply:
Tbh, I don't think we have too. I don't see future implementation where this method is called twice within the same execution.